templates/expo-template-tabs/navigation/useWebLinking.js

```diff
@@ -0,0 +1,33 @@
+import { useMemo } from 'react';
+import { Platform } from 'react-native';
+import { useLinking } from '@react-navigation/native';
+
+export default function useWebLinking(containerRef) {
+  if (Platform.OS === 'web') {
+    const { getInitialState } = useLinking(containerRef, {
+      prefixes: [],
+      config: {
+        HomeStack: {
+          path: 'home-stack',
```

Comment:
You don't usually want type of navigator in the URL. Let's remove the suffix. Maybe will be easier if we had different names for the stack and the screen inside.

Seems that in the original template, `path` was set to empty string. Wojtek is working on this functionality.

Reply:
this path is actually not changing anything, because the link refers to the most nested screen, so it's always `/links/ `/screen` etc.

Reply:
You can still go to the URL `/home-stack` to open a specific stack